modules/gapi/doc/slides/get_sty.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+MTHEME_VER=2fa6084b9d34fec9d2d5470eb9a17d0bf712b6c8
+MTHEME_DIR=mtheme.sty
+
+function make_sty {
+    if [ -d "$MTHEME_DIR" ]; then rm -rf "$MTHEME_DIR"; fi
+    mkdir "$MTHEME_DIR"
+
+    # Download template from Github
+    tmp_dir=$(mktemp -d)
+    wget -P "$tmp_dir" -c https://github.com/matze/mtheme/archive/${MTHEME_VER}.tar.gz
+    pushd "$tmp_dir"
+    tar -xzvf "$MTHEME_VER.tar.gz"
+    popd
+    make -C "$tmp_dir"/mtheme-"$MTHEME_VER"
+    cp   -v "$tmp_dir"/mtheme-"$MTHEME_VER"/*.sty "$MTHEME_DIR"
+    rm -r "$tmp_dir"
```

Comment:
Let's make git happy in case of branch switching:
```
echo "*" > "$MTHEME_DIR/.gitignore"
```

Reply:
nice

Reply:
yeah but do you think it makes sense to put this `mtheme.sty/.gitignore` under a regular source control? I believe it will show as "untracked" otherwise

Reply:
well, it works even without that